src/implementations/vanilla/src/basics/text-link/tests/tests-text-link.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../dist/hig.css">
+        <script type="text/javascript" src="../../../../dist/hig.js"></script>
+        <style>
+            .tests-text-link {
+                padding: 40px;
+                margin: 0;
+                display: flex;
+                justify-content: center;
+            }
+        </style>
+    </head>
+    <body>
+        <div class='tests-text-link tests-text-link-defaults'></div>
+        <div class='tests-text-link tests-text-link-with-href'></div>
+        <div class='tests-text-link tests-text-link-secondary'></div>
+        <div class='tests-text-link tests-text-link-secondary-with-href'></div>
+        <div class='tests-text-link tests-text-link-with-onclick'></div>
+        <div class='tests-text-link tests-text-link-with-onclick-and-href'></div>
+        <script>
+
+            // ----------------------------
+            // DEFAULT TEXTLINK
+            // ----------------------------
+
+            var textLinkWithDefaults = new Hig.TextLink();
+            textLinkWithDefaults.mount('.tests-text-link-defaults');
+
+            // ----------------------------
+            // TEXTLINK WITH LINK
+            // ----------------------------
+
+            var textLink = new Hig.TextLink({
+                href: 'https://github.com/Autodesk/hig',
+                text: 'This is a TextLink to HIG repository url'
+            });
+            textLink.mount('.tests-text-link-with-href');
+
+            // ----------------------------
+            // SECONDARY TEXTLINK
+            // ----------------------------
+
+            var secondaryTextLink = new Hig.TextLink({
+                text: 'This is a secondary TextLink',
```

Comment:
similarly is this missing the default "href" attribute on purpose?

Reply:
(see above comment)

Reply:
got it.